examples/customer-satisfaction-survey-analysis/Zendesk customer satisfaction survey analysis.md

```diff
@@ -0,0 +1,30 @@
+# Zendesk customer satisfaction survey analysis
+
+This use case demonstrates how the Zendesk REST API v2 can be utilized to improve customer service quality by 
+analyzing feedback from satisfaction surveys sent after resolving support tickets. The example involves a sequence 
+of actions that leverage the Zendesk REST API to automate and streamline these efforts.
+
+## Prerequisites
+
+### 1. Setup Zendesk account
+
+Refer to the [Setup Guide](https://central.ballerina.io/ballerinax/zendesk/latest#setup-guide) to set up your zendesk
```

Comment:
`Setup Guide` heading uses sentence case, and here we have used title case, if this is not an issue, just ignore it.

Reply:
+1 to use the sentence case in here as well. Updated with https://github.com/ballerina-platform/module-ballerinax-zendesk/pull/3/commits/828c439bd19703622914569cbaed4b46095e3e97